specification/communication/data-plane/CallingServer/preview/2021-06-15-preview/communicationservicescallingserver.json

```diff
@@ -0,0 +1,1467 @@
+            },
+            "x-ms-error-response": true
+          }
+        }
+      }
+    },
+    "/calling/serverCalls/{serverCallId}/:playAudio": {
+      "post": {
+        "tags": [
+          "ServerCall"
+        ],
+        "summary": "Play audio in a call.",
+        "description": "Play audio in a call.",
+        "operationId": "ServerCall_PlayAudio",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "serverCallId",
+            "in": "path",
+            "description": "The server call id.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "request",
+            "in": "body",
+            "description": "Play audio request.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/PlayAudioRequest"
+            }
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Returns the play audio response.",
+            "schema": {
+              "$ref": "#/definitions/PlayAudioResult"
+            }
+          },
+          "401": {
+            "description": "Unauthorized",
+            "schema": {
+              "$ref": "#/definitions/CommunicationError"
+            },
+            "x-ms-error-response": true
+          },
+          "400": {
+            "description": "Bad request",
+            "schema": {
+              "$ref": "#/definitions/CommunicationError"
+            },
+            "x-ms-error-response": true
+          },
+          "500": {
+            "description": "Internal server error.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationError"
+            },
+            "x-ms-error-response": true
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CallModality": {
+      "enum": [
+        "audio",
+        "video"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "CallModality",
+        "modelAsString": true
+      }
+    },
+    "EventSubscriptionType": {
+      "enum": [
+        "participantsUpdated",
+        "dtmfReceived"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "EventSubscriptionType",
+        "modelAsString": true
+      }
+    },
+    "CreateCallRequest": {
+      "description": "The request payload for create call.",
+      "required": [
+        "targets",
+        "source",
+        "callbackUri"
+      ],
+      "type": "object",
+      "properties": {
+        "alternateCallerId": {
+          "$ref": "#/definitions/PhoneNumberIdentifier",
+          "description": "The alternate identity of the source of the call if dialing out to a pstn number"
+        },
+        "targets": {
```

Comment:
I see that we are also using the term "participants" when inviting more parties to the call. We should find a consistent term to use (I know I mentioned "to" or "destination" in the in-person review/overview earlier, but that may not make as much sense outside the context of making the call)

Reply:
We need to think about this to see if we can find a better name. For public preview can we go with what we have currently? Client Calling SDK is also using "addParticipant" for this. https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/calling-client-samples?pivots=platform-web